leetcode/69.c

```diff
@@ -0,0 +1,31 @@
+//method-1
+
+
+
+//According to the problem statement we must not use any "built-in exponent function" so by considering that...
+
+int mySqrt(int x){
+
+//a new variable which runs till we reach the exact number or the number which is just exactly greater than the number(x).
+  long long int k=1;
+  while(k*k <= x ){
```

Comment:
I believe using O(n) there it's too long. You could use the binary search approach.